c/godwoken.mol

```diff
@@ -117,22 +118,19 @@ vector DepositRequestVec <DepositRequest>;
 
 struct RawWithdrawalRequest {
```

Comment:
We should update https://github.com/nervosnetwork/godwoken/blob/master/docs/life_of_a_godwoken_transaction.md#withdraw later.

Reply:
Right